README.md

```diff
@@ -1,92 +1,224 @@
+
+
 <!--
 ******************************************************************************
 * Copyright 2014-2020 Intel Corporation
-*
+  *
```

Comment:
New alignment looks strange - previous one looks better

Reply:
Thanks. I fixed it